drools-compiler/src/main/java/org/drools/compiler/kproject/models/KieModuleModelImpl.java

```diff
@@ -211,13 +213,15 @@ public void onNode(HierarchicalStreamReader reader, String name, String value) {
     }
 
     private static class KieModuleValidator {
-        private static final Schema schema = loadSchema();
+        private static final Schema schema = loadSchema("org/kie/api/kmodule.xsd");
+        private static final Schema oldSchema = loadSchema("org/kie/api/old-kmodule.xsd");
```

Comment:
`old-kmodule.xsd` is not deployed in `kie-api-6.0.0.Final.jar` - I guess it came later, because the comment down there says 
`// For backwards compatibility, validate against the old namespace (which has 6.0.0 hardcoded)`

I guess we can remove this entire oldSchema fallback part as it could run into Exceptions due to the missing xsd file